### PR TITLE
Aztec Revivalist FS/ Name cleanup and reorganization

### PR DESCRIPTION
**Pull request by megas4o**
Aztec Revivalist FS. Most of the cosmetic stuff is done, still needs work under the hood. Name duplicate cleanup and alphabetizing

### CODE REVIEW

File: src/npc/fFeelings.tw [hunk 1]
```diff
@@ -726,6 +726,15 @@ My favorite part of my body i<<s>>
 		Being a <<s>>lave in the new Rome is a little <<s>>cary, <<Master>>. I hear <<s>>lave<<s>> fighting <<s>>ometime<<s>>.
 	<</if>>
 <</if>>
+<<if $arcologies[0].FSAztecRevivalist >= 10>>
+	<<if ($activeSlave.intelligence > 1)>>
+		The new Aztec Empire is enthralling, <<Master>>. I'm amazed at how easy jump to sacrifice and debauchery when they're offered.
```
Comment (DarkTalon25) [inline src/npc/fFeelings.tw]: Who is "they"?

File: src/uncategorized/assistantEvents.tw [hunk 2]
```diff
@@ -181,6 +185,8 @@ of your personal assistant pops up on the nearest screen.  "<<if $PC.title != 0>
 		adjusts her outfit to resemble a crusader of your new religion.
 	<<elseif $arcologies[0].FSRomanRevivalist != "unset">>
 		adjusts her outfit to resemble a Roman warrior to conform with your vision of building a new Rome.
+	<<elseif $arcologies[0].FSAztecRevivalist != "unset">>
+		adjusts her outfit to resemble a Aztec champion to conform with your vision of building a new Rome.
```
Comment (DarkTalon25) [inline src/uncategorized/assistantEvents.tw]: It should be "a new Aztec Empire" here.

File: src/uncategorized/assistantEvents.tw [hunk 3]
```diff
@@ -257,6 +265,8 @@ of your personal assistant pops up on the nearest screen.  "<<if $PC.title != 0>
 		adjusts appearance to resemble an angel of your new religion.
 	<<elseif $arcologies[0].FSRomanRevivalist != "unset">>
 		adjusts her outfit to resemble a Roman lady to conform with your vision of building a new Rome.
+	<<elseif $arcologies[0].FSAztecRevivalist != "unset">>
+		adjusts her outfit to resemble a Aztec woman to conform with your vision of building a new Rome.
```
Comment (DarkTalon25) [inline src/uncategorized/assistantEvents.tw]: And here.

File: src/uncategorized/assistantEvents.tw [hunk 4]
```diff
@@ -295,6 +305,8 @@ of your personal assistant pops up on the nearest screen.  "<<if $PC.title != 0>
 		adjusts her outfit to prominently display a religious symbol of your new religion.
 	<<elseif $arcologies[0].FSRomanRevivalist != "unset">>
 		adjusts her outfit to resemble a proper upper-class Roman lady to conform with your vision of building a new Rome.
+	<<elseif $arcologies[0].FSAztecRevivalist != "unset">>
+		adjusts her outfit to resemble a high standing Aztec priestess to conform with your vision of building a new Rome.
```
Comment (DarkTalon25) [inline src/uncategorized/assistantEvents.tw]: Here too.

File: src/uncategorized/economics.tw [hunk 5]
```diff
@@ -630,6 +630,29 @@ On formal occasions, you are announced as $PCTitle.
...
+<<elseif $arcologies[0].FSAztecRevivalist != "unset">>
+	<<if $PC.age >= 2>>
+		Society @@.green;approves@@ of your advancing age, which advances the ancient Aztec ideal an experienced leader to the people.
+		<<FSChange "AztecRevivalist" 1>>
+	<</if>>
+	<<if $HeadGirl == 0>>
+		Society @@.red;disapproves@@ of you not having a Head Girl as an advisor and assistant.
+		<<FSChange "AztecRevivalist" -2>>
+	<<else>>
+		Society @@.green;approves@@ of your reliance on a Head Girl, as an advisor and assistant.
```
Comment (DarkTalon25) [inline src/uncategorized/economics.tw]: Unnecessary comma.

File: src/uncategorized/economics.tw [hunk 6]
```diff
@@ -630,6 +630,29 @@ On formal occasions, you are announced as $PCTitle.
...
+		<<FSChange "AztecRevivalist" 2>>
+	<</if>>
+	<<if $PC.warfare < 50>>
+		Society @@.red;disapproves@@ of you not being properly trained in the art of combat.
+		<<FSChange "AztecRevivalist" -2>>
+	<<else>>
+		Society @@.green;approves@@ of having a leader, than can acquaint themselves properly in a life threatening situation.
```
Comment (DarkTalon25) [inline src/uncategorized/economics.tw]: Unnecessart comma.

*that

I'd say that "defend" is a more suitable word than "acquaint".

File: src/uncategorized/economics.tw [hunk 7]
```diff
@@ -887,6 +910,7 @@ __Arcology Management__
 $ACitizens citizens, $ASlaves sex slaves, and <<print $AHelots+$helots>> menial slaves live in your arcology.
 <<if $arcologies[0].FSSupremacistLawME == 1>>The citizenry is entirely $arcologies[0].FSSupremacistRace.<</if>>
 <<if $arcologies[0].FSRomanRevivalistLaw == 1>>The citizens take pride in their martial duties, preferring to wear utilitarian clothing even when off duty.<</if>>
+<<if $arcologies[0].FSAztecRevivalistLaw == 1>>Most citizens wear satin loincloths and cloakes, distinguised from most slaves only by the richness of their adornments.<</if>>
```
Comment (DarkTalon25) [inline src/uncategorized/economics.tw]: *cloaks

File: src/uncategorized/assistantEvents.tw (hunk 2 above)
Comment (Forovakill) [inline src/uncategorized/assistantEvents.tw]: Also, it should be **an** Aztec champion.

File: src/uncategorized/assistantEvents.tw (hunk 3 above)
Comment (Forovakill) [inline src/uncategorized/assistantEvents.tw]: **an** Aztec woman

File: src/npc/fFeelings.tw (hunk 1 above)
Comment (Forovakill) [inline src/npc/fFeelings.tw]: Perhaps it should be "I'm amazed at how easy _people_ jump to sacrifice and debauchery when they're offered."

**Comment by freecitiesdev** [comment]
Don't worry, I haven't forgotten about this. There's been some broad spectrum violence to the code lately and once it settles down I'll review this again. FS code is scattered so broadly that I'm saving this for an update that's a little more sedate.

**Comment by freecitiesdev** [comment]
megas4o, are you up for resolving these conflicts? If not, I'm willing to do it, but it'll drop down the priority list slightly since there are serious bugs to be addressed right now in other areas.

**Pull request closed without merging**